tutorials/Augurpy.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    
</pre><p>I'd suggest to delete the unused empty lines :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/augurpy/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>